tests/test_demes.py

```diff
@@ -367,6 +367,24 @@ def test_pulses_ordering(self):
         assert d.events[3].dest == "A"
         assert d.events[3].proportion == 0.4
 
+    def test_multipulse(self):
+        b = demes.Builder(defaults=dict(epoch=dict(start_size=1)))
+        b.add_deme("a")
+        b.add_deme("b")
+        b.add_deme("c")
+        b.add_pulse(sources=["a", "b"], proportions=[0.1, 0.2], dest="c", time=1)
+        g = b.resolve()
+        d = msprime.Demography.from_demes(g)
+        assert len(d.events) == 2
+        assert d.events[0].__class__ == msprime.MassMigration
```

Comment:
```suggestion
        assert isinstance(d.events[0], msprime.MassMigration)
```

Reply:
isinstance is the idiomatic way of doing this, comparing to ``__class__`` can be brittle